### PR TITLE
Add start angle

### PR DESCRIPTION
**Pull request by billingsley-john**
## Proposed changes

PR which adds a start angle to CoolantChannelRing* parametric components.

## Types of changes

What types of changes does your code introduce to the Paramak?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [x] New tests

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Pep8 applied
- [x] Unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

No further comments

### CODE REVIEW

File: tests/test_parametric_components/test_CoolantChannelRingStraight.py [hunk 1]
```diff
@@ -55,5 +56,17 @@ def test_CoolantChannelRingStraight_volume(self):
         assert self.test_shape.volume == pytest.approx(
             math.pi * (10 ** 2) * 100 * 8)
 
+    def test_CoolantChannelRingStraight_start_angle(self):
```
Comment (Shimwell) [inline tests/test_parametric_components/test_CoolantChannelRingStraight.py:59]:
```suggestion
    def test_start_angle(self):
```

File: tests/test_parametric_components/test_CoolantChannelRingCurved.py [hunk 2]
```diff
@@ -45,5 +46,17 @@ def test_CoolantChannelRingCurved_relative_volumes(self):
         self.test_shape.force_cross_section = True
         assert test_volume < self.test_shape.volume
 
+    def test_CoolantChannelRingCurved_start_angle(self):
```
Comment (Shimwell) [inline tests/test_parametric_components/test_CoolantChannelRingCurved.py:49]:
```suggestion
    def test_start_angle(self):
```

**Review by Shimwell — APPROVED** [review]
optional, you can change from 8 coolant channels which will help with the code inspector score

**Comment by Shimwell** [comment]
the failing test is related to the neutronics tests and will be been fixed by #600 . So nothing to worry about here

**Comment by billingsley-john** [comment]
New version in #607

**Pull request closed without merging**